config.m4

```diff
@@ -230,8 +236,40 @@ if test "$PHP_MONGODB" != "no"; then
     AC_DEFINE(HAVE_SYSTEM_LIBMONGOC, 1, [Use system libmongoc])
   fi
 
+  if test "$PHP_CLIENT_SIDE_ENCRYPTION" != "no" -a "$PHP_LIBBSON" = "yes"; then
+    AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+    AC_MSG_CHECKING(for libmongocrypt)
+
+    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongocrypt-1.0; then
+      if $PKG_CONFIG libmongocrypt-1.0 --atleast-version 1.0.0; then
+        PHP_MONGODB_MONGOCRYPT_CFLAGS=`$PKG_CONFIG libmongocrypt-1.0 --cflags`
+        PHP_MONGODB_MONGOCRYPT_LIBS=`$PKG_CONFIG libmongocrypt-1.0 --libs`
+        PHP_MONGODB_MONGOCRYPT_VERSION=`$PKG_CONFIG libmongocrypt-1.0 --modversion`
+        AC_MSG_RESULT(version $PHP_MONGODB_MONGOCRYPT_VERSION found)
+
+        PHP_MONGODB_CFLAGS="$PHP_MONGODB_CFLAGS $PHP_MONGODB_MONGOCRYPT_CFLAGS"
+        PHP_EVAL_LIBLINE($PHP_MONGODB_MONGOCRYPT_LIBS, MONGODB_SHARED_LIBADD)
+        AC_DEFINE(HAVE_SYSTEM_LIBMONGOCRYPT, 1, [Use system libmongocrypt])
+        AC_SUBST(MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION, 1)
+      elif test "$PHP_CLIENT_SIDE_ENCRYPTION" = "yes"; then
+        AC_MSG_ERROR(system libmongocrypt must be upgraded to version >= 1.0.0)
+      else
+        AC_SUBST(MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION, 0)
+        AC_MSG_RESULT(found in an older version, compiling without client-side encryption)
+      fi
+    else
+      if test "$PHP_CLIENT_SIDE_ENCRYPTION" = "yes"; then
+        AC_MSG_ERROR(pkgconfig and libmongocrypt must be installed)
+      else
+        AC_SUBST(MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION, 0)
+        AC_MSG_RESULT(not found, compiling without client-side encryption)
+      fi
+    fi
+  fi
+
   if test "$PHP_LIBBSON" = "no" -a "$PHP_LIBMONGOC" = "no"; then
     PHP_MONGODB_BUNDLED_CFLAGS="$STD_CFLAGS -DBSON_COMPILATION -DMONGOC_COMPILATION"
+    PHP_MONGODB_LIBMONGOCRYPT_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -std=gnu99"
```

Comment:
That was the only standard that got this to compile on evergreen. `-std=c99` did not work, I explicitly had to choose gnu99. Without this, the build failed because of inline declarations in `for` loops, which were only added in C99: `for (int i = 0; ...)`. c99 brought on other errors, while gnu99 worked successfully. Not sure what the difference is, but happy to change to a different standard as long as it keeps the build successful.

Reply:
I think libmongocrypt should ideally build under `-std=c89`. I created https://jira.mongodb.org/browse/MONGOCRYPT-219 to make any necessary fixes. If this is something we should do now, I can schedule it.

Reply:
Is there any risk for us building with `-std=gnu99`? `c99` doesn't support `ssize_t`, which is why that doesn't work. We just need to ensure this works on all supported platforms.

Reply:
I don't know how portable `gnu99` is, which is why I originally though to suggest `AC_PROG_CC_STDC`. I suppose we can keep this for now and see if any users complain after the release. I would ultimately like to see MONGOCRYPT-219 addressed, but I think we can wait on that vs. other higher priority work.

@alcaeus would you mind leaving a TODO comment above this with a link to MONGOCRYPT-219? Then I think we can resolve this thread.

Reply:
I don't know how portable `gnu99` is, which is why I originally though to suggest `AC_PROG_CC_STDC`. I suppose we can keep this for now and see if any users complain after the release. I would ultimately like to see MONGOCRYPT-219 addressed, but I think we can wait on that vs. other higher priority work.

@alcaeus would you mind leaving a TODO comment above this with a link to MONGOCRYPT-219? Then I think we can resolve this thread.

Reply:
I had added this but forgot to move it around when moving the definition of `PHP_MONGODB_LIBMONGOCRYPT_CFLAGS`. This is fixed now.